src/test/java/com/google/api/codegen/testdata/ruby_json_library.baseline

```diff
@@ -1,4 +1,4 @@
-============== file: library_service_client_config.json ==============
+============== file: google/example/library/v1/library_service_client_config.json ==============
```

Comment:
Well, this does not look correct.  Probably not by this CL, but something went wrong during some refactoring...

Ruby code should be generated under lib/ directory. So this is lib/google/example/library/v1/library_service_client_config.json (and library_service_api.rb place is wrong right now)